mitthesis.cls

```diff
@@ -529,16 +520,10 @@ requirements for the \@degreeword\ of \\
 \def\MIT{MASSACHUSETTS INSTITUTE OF TECHNOLOGY}
 \def\Mit{Massachusetts Institute of Technology}
 
-%bkettle 2023: thesis permission text has changed
-\def\permission{\par\noindent{\centering
-  The author hereby grants to MIT a nonexclusive, worldwide, 
-  irrevocable, royalty-free license to exercise any and all
-  rights under copyright, including to reproduce, preserve, 
-  distribute and publicly display copies of the thesis, or 
-  release the thesis under an open-access license.}\par}
+\def\permission{\par\noindent{\centering\normalsize
+   The author hereby grants to MIT a nonexclusive, worldwide, irrevocable, royalty-free license to exercise any and all rights under copyright, including to reproduce, preserve, distribute and publicly display copies of the thesis, or release the thesis under an open-access license.}\par}
 
-\def\signature#1#2{\par\noindent#1\dotfill\null\\*
-  {\raggedleft #2\par}}
+\def\signature#1#2{\normalsize\raggedright\parbox{7em}{#1:}\parbox[t]{\dimexpr\textwidth-7em\relax}{#2\par}}
```

Comment:
If we want to keep using choosecase, it could also go in the definition of signature here.

As an aside, this signature definition is way cleaner. Thanks for taking the time to set that up.